TESTS/mbed_hal/watchdog_timing/main.cpp

```diff
@@ -134,9 +134,9 @@ int testsuite_setup(const size_t number_of_cases)
 
 Case cases[] = {
     Case("Timing, 200 ms", case_setup, test_timing<200UL, 55UL>),
-    Case("Timing, 500 ms", case_setup, test_timing<500UL, 65UL>),
-    Case("Timing, 1000 ms", case_setup, test_timing<1000UL, 130UL>),
-    Case("Timing, 3000 ms", case_setup, test_timing<3000UL, 190UL>),
+    Case("Timing, 500 ms", case_setup, test_timing<500UL, 130UL>),
```

Comment:
Letting this go through for now, but do we know why these timing are supposedly too tight?

Reply:
Unfortunately not; due to the nature of this test, the reason might be:
* slow UART performance on this particular board,
* actual watchdog timer inaccuracy.